src/Apps/Sell/Components/BottomFormNavigation.tsx

```diff
@@ -2,10 +2,17 @@ import { Box, Button, Flex } from "@artsy/palette"
 import { useSellFlowContext } from "Apps/Sell/SellFlowContext"
 import createLogger from "Utils/logger"
 import { useFormikContext } from "formik"
+import { FC } from "react"
 
 const logger = createLogger("BottomFormNavigation.tsx")
 
-export const BottomFormNavigation = () => {
+interface BottomFormNavigationProps {
+  loading?: boolean
```

Comment:
Can we move this loading state into the context (e.g. as `isAssetUploading`) or retrieve the state from Formik (e.g. `form.photos.some(photo => photo.loading)`)?

Reply:
And we might want to at least disable the back button in this case as well to avoid unpredictable behavior.